spec/index.html

```diff
@@ -1064,9 +1089,13 @@ <h3>RDF Dataset Comparison</h3>
         the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in |DG2|.</li>
       <li>The [=named graph=] (|n|, |G|) is in |NG1| if and only if
         there is a [=named graph=] (<var>n'</var>, <var>G'</var>) in |NG2| such that
-        |M|(|n|) = <var>n'</var> and
-        the triple (|s|, |p|, |o|) is in |G| if and only if
-        the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in <var>G'</var>.</li>
+        the following are true:
+        <ul>
+          <li>|M|(|n|) is [=RDF term equality|equal=] to <var>n'</var>.</li>
+          <li>The triple (|s|, |p|, |o|) is in |G| and
+            the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in <var>G'</var>.</li>
```

Comment:
```suggestion
          <li>The triple (|s|, |p|, |o|) is in |G| if and only if
            the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in <var>G'</var>.</li>
```
I assume that the change of "if and only if" to "and" was not intended.

Reply:
No, I intentionally changed the `if and only if` to `and`. Try reading the conditional from the start with less markup, and leaving out the first, simple bullet item from the `<ul>` —
```
The named graph (n, G) is in NG1 if and only if
   there is a named graph (n', G') in NG2 such that
      the triple (s, p, o) is in G if and only if
         the triple ( M(s), M(p), M(o) ) is in G'.
```
— Vs. my simple `and` —
```
The named graph (n, G) is in NG1 if and only if
   there is a named graph (n', G') in NG2 such that
      the triple (s, p, o) is in G and
         the triple ( M(s), M(p), M(o) ) is in G'.
```